release-notes/next.yml

```diff
@@ -1,4 +1,8 @@
 releases:
+  37.20221021.1.1:
+    issues:
+      - text: "HIGH OpenSSL vulnerabilities (CVE-2022-3786 and CVE-2022-3602)"
```

Comment:
Can we rename that issue (and this text entry) to something like:

> Update OpenSSL for CVE-2022-3786 and CVE-2022-3602

?

Reply:
done